Jenkinsfile_CNP

```diff
@@ -101,7 +99,7 @@ withPipeline(type, product, component) {
       allowMissing         : true,
       alwaysLinkToLastBuild: true,
       keepAll              : true,
-      reportDir            : "test-output/functional/reports/allure",
+      reportDir            : "test-output/functional/allure",
```

Comment:
I saw u changed the report structure. It is fine here for CNP but u need to change the nightly build full functional path too to remove the "reports" part from the path

Reply:
Resolved in https://github.com/hmcts/idam-user-dashboard/pull/115/commits/037bacaa6471e1b2268eab0aaaa9fa512a17e4f6 :)